modules/sandboxed-containers-preparing-openshift-cluster.adoc

```diff
@@ -3,5 +3,105 @@
+
+* Your cluster must be installed on bare metal infrastructure with {op-system-first} workers.
+// To Do: Add link to bare metal instructions: ../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal]
++
+[IMPORTANT]
+====
+* {sandboxed-containers-first} only supports {op-system} worker nodes. RHEL 7 or RHEL  8 nodes are not supported.
+* Nested virtualization is not supported.
+====
+* Your cluster must use installer-provisioned infrastructure.
+// To Do: Add link to installer provisioned infrastructure: ../../installing/installing_bare_metal_ipi/ipi-install-overview.adoc#ipi-install-overview[Installer provisioned infrastructure]
+
+[id="sandboxed-containers-additional-resource-requirements_{context}"]
+== Additional resource requirements for {sandboxed-containers-first}
+
+{sandboxed-containers-first} is a product that brings the ability to run workloads inside a sandboxed runtime, such as Kata Containers, to your {product-title} clusters. Each pod is represented by a virtual machine (VM). Each VM runs in a `qemu` process and hosts a `kata-agent` process that acts as a supervisor for managing container workloads and processes that are running in those containers. There are two additional processes that add more overhead:
+
+* `containerd-shim-kata-v2` is used to communicate with the pod.
+* `virtiofsd` handles host file system access on behalf of the guest.
+
+Each VM is configured with a default amount of memory. Additional memory is hot-plugged into the VM for containers that explicitly request memory.
+
+* If a container runs without a given memory resource, it is able to consume free memory. It will do so until the total memory used by the VM reaches the default allocation. The guest and its I/O buffers also consume memory.
+* If a container is given a specific amount of memory, then that memory is hot-plugged into the VM before the container starts.
+* If a memory limit is specified, then the workload is terminated if it consumes more memory than the limit. If no memory limit is specified, the kernel  that is running on the virtual machine might run out of memory. If the kernel runs out of memory it might terminate other process on the virtual machine.
+
+.Indicative memory sizes
+
+The table below lists some typical values for resource allocation.
+
+[cols="2,2"]
+|===
+|Resource |Value
+
+|Memory allocated by default to a virtual machine | 2Gi
+|Guest Linux kernel memory usage at boot | ~110Mi
+|Memory used by the QEMU process (excluding VM memory) | ~30Mi
+|Memory used by the `virtiofsd` process (excluding VM I/O buffers) | ~10Mi
+|Memory used by the `containerd-shim-kata-v2` process | ~20Mi
+|File buffer cache data after running `dnf install` on Fedora | ~300Mi* ^[1]^
+|===
+[.small]
+--
+
+1. File buffers appear and are accounted for in multiple locations:
+* In the guest where it appears as file buffer cache.
+* In the `virtiofsd` daemon, mapped allow user-space file I/O operations.
```

Comment:
Probably my ignorance. I don't know what _mapped allow user-space file I/O operations_ means. It just seems incorrect